### DISCUSSION TITLE
Adding inkbird IHC-200

### DISCUSSION THREAD

**Discussion by DoumP** [make-all/tuya-local#572] [Q&A] [answered]
Hi,
I'm trying to add a new device (inkbird IHC-200) which is similar to the inkbird ITC-308 but with humidity sensor instead of temperature.
I've started a yaml file here: https://github.com/DoumP/tuya-local/blob/main/custom_components/tuya_local/devices/inkbird_ihc200_hygrostat.yaml
So far I can see the current humidity and I can set it but I would like it to look like the ITC-308 does in one controler, now I'm getting two separate tile and I have no clue why. Note that I'm just starting with HA and tuya so I'm still missing a lot of specific understanding for those products. 
Here's a screenshot, you can see the itc-308 with one tile to see/control the temperature but the thc-200 is showing the humidity in a separate tile than the control.
![image](https://user-images.githubusercontent.com/10068649/228339407-1d70babc-3c60-476b-9b26-260a139d1b80.png)
Any help or pointer to documentation would be appreciated.

**Comment by make-all** [comment, marked as answer]
That is just the way Home Assistant does it.
The climate platform is older, from a time when they thought one device = one entity was the way to do it, and bundled all the functionality of the most complex air conditioning unit into the climate platform, and the corresponding climate card.
But by the time they added the humidifier platform, the philosophy had changed to have simpler entities, and separate sensors for different measurements. The UI card followed the underlying entity, but you may be able to find a custom card that combines the humidity entity with the current humidity sensor to look more like the climate card.

**Reply by DoumP** [reply]
Thank you, I'll look into that.
